dashboard/test/controllers/api_controller_test.rb

```diff
@@ -245,6 +245,8 @@ def make_progress_in_section(script)
   end
 
   test "should get surveys for section with script with anonymous level_group assessment" do
+    skip "temporarily disabled on CircleCI after Ubuntu 14 upgrade" if ENV['CI']
```

Comment:
It looks like the failure is the result of array elements being reordered. Is order important (in which case why are we disabling a failing test)? Is order unimportant (in which case why don't we simply permute the elements in the expected response)?